src/passes/Print.cpp

```diff
@@ -170,6 +170,8 @@ struct PrintSExpression : public UnifiedExpressionVisitor<PrintSExpression> {
 
   std::vector<HeapType> heapTypes;
 
+  unsigned lastPrintIndent = 0;
```

Comment:
Please add a comment explaining the motivation for tracking the print indent, maybe with an example? I am imagining something like this:
```cpp
//  Track the print indent so that we can see when it changes. That affects how we print debug annotations. In particular, we don't want to print repeated debug locations for children, like this:
//
//  ;;@ file.cpp:20:4
// (block
//    ;; no need to annotate here
//    (nop)
//
// But we do want to print an annotation even if it repeats if it is not a child:
//
//  ;;@ file.cpp:20:4
// (block
// )
// ;;@ file.cpp:20:4 - this is worth annotating
// (nop)
```